crates/host/src/lib.rs

```diff
@@ -73,16 +74,29 @@ impl<'a> TryFrom<&'a str> for ResourceRef<'a> {
                         .to_file_path()
                         .map(Self::File)
                         .map_err(|()| anyhow!("failed to convert `{url}` to a file path")),
-                    "bindle" => s
-                        .strip_prefix("bindle://")
-                        .map(Self::Bindle)
-                        .context("invalid Bindle reference"),
-                    "oci" => s
-                        .strip_prefix("oci://")
-                        .map(Self::Oci)
-                        .context("invalid OCI reference"),
-                    // TODO: Support other schemes
-                    scheme => bail!("unsupported scheme `{scheme}`"),
+                    "bindle" => {
+                        // Note: bindle is not a scheme, but using this as a prefix takes out the guesswork
+                        s.strip_prefix("bindle://")
+                            .map(Self::Bindle)
+                            .context("invalid Bindle reference")
+                    }
+                    "oci" => {
+                        // Note: oci is not a scheme, but using this as a prefix takes out the guesswork
+                        s.strip_prefix("oci://")
+                            .map(Self::Oci)
+                            .context("invalid OCI reference")
+                    }
+                    scheme @ ("http" | "https") => {
+                        debug!(%url, "interpreting reference as OCI");
+                        s.strip_prefix(&format!("{scheme}://"))
+                            .map(Self::Oci)
+                            .context("invalid OCI reference")
+                    }
+                    _ => {
+                        // handle strings like `registry:5000/v2/foo:0.1.0`
```

Comment:
The ref in the comment should already be handled by the `Err` case below 

Can you give an example of a ref that would be parsed here?